spec/features/19_admin_deletes_user_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+feature "Admin can update word" do
+  let!(:admin) { FactoryGirl.create(:admin) }
+  let!(:users) { FactoryGirl.create_list(:user, 4) }
+  let!(:words) { FactoryGirl.create_list(:word, 4, user: users[0]) }
+  let!(:safe_word) { FactoryGirl.create(:word, user: users[3]) }
+  let!(:safe_review) { FactoryGirl.create(:review, user: users[2], word: safe_word) }
+  let!(:reviews) { FactoryGirl.create_list(:review, 4, user: users[0], word: words[0]) }
+  let!(:reviews) { FactoryGirl.create_list(:review, 4, user: users[0], word: safe_word) }
+
+  context "As an admin" do
+    scenario "I can delete a user from the user's profile page" do
+      login_as(admin, scope: :user)
+      visit user_path(users[0])
+      deleted_username = users[0].username
```

Comment:
You can do this in a `let` too - `lets` can hold any arbitrary code. 

`let!(:deleted_username) { users.first.username}`